tests/test__default.py

```diff
@@ -447,7 +447,9 @@ def app_identity(monkeypatch):
     yield app_identity_module
 
 
-def test__get_gae_credentials(app_identity):
+@mock.patch.dict(os.environ)
+def test__get_gae_credentials_gen1(app_identity):
+    os.environ[environment_vars.LEGACY_APPENGINE_RUNTIME] = "python27"
```

Comment:
Tests below (`test__get_gae_credentials_no_app_engine`, `test__get_gae_credentials_no_apis`) also need to set this envvar:  otherwise, the code they are supposed to cover isn't reached, due to the early exit.

Reply:
Done.  Testing both with and without the env var in those tests for completeness.

FYI in the environments where that env var is normally set, the GAE APIs would always be available.  Someone would have to be explicitly setting APPENGINE_RUNTIME="python27" outside of a gae/devappserver environment to reach that codepath.  That said, they could be doing that, so test added.